civiform/civiform
# Create North Star "Prevent Duplicate Submission" page

### Description

<img width="1435" alt="Screenshot 2024-07-03 at 9 35 49 AM" src="https://github.com/civiform/civiform/assets/23390490/f2a6292b-d871-4e41-8f38-13e656c2d89b">

## Release notes

n/a

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1. Submit an application to a program
2. Click that program again
3. Submit an application without changing anything
4. Expect to see the "Prevent Duplicate Submission" page

### Issue(s) this completes

Fixes #7875
